dataset/dictionary/organizations.json5

```diff
@@ -1,11 +1,8 @@
 [
   {
-    en: "",
+    en: "The Moles",
     ja: "",
```

Comment:
Can you remove this for now? It doesn't raise errors on the test even if there is no Japanese.
```suggestion
```

Reply:
You mean remove the whole word or just remove the ja line

Reply:
Oh, sorry for the unclearness.
Just remove `ja` line. Keep English and Chinese.